<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemSearch</class>
 <widget class="QDialog" name="ItemSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_Items">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>731</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Search">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>22</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_ItemCode">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>22</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Code:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Search">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
